kustomize/monitoring/base/monitoring.yaml

```diff
@@ -636,13 +636,14 @@ data:
           "fiscalYearStartMonth": 0,
           "graphTooltip": 0,
           "id": null,
-          "iteration": 1651813612530,
+          "iteration": 1652333212917,
```

Comment:
Something hardcoded here?

Reply:
No its the number of time Ank might have executed some dashboard

Reply:
@songole This will get changed whenever we make changes to the dashboard, no issues with this.